packages/governance/test/swingsetTests/contractGovernor/add-governedContract.js

```diff
@@ -0,0 +1,194 @@
+ *   interlockPowers} powers
+ * @param {{
+ *   options: {
+ *     contractInstallation: Installation;
+ *     producedKey: string | undefined;
+ *     label: string | undefined;
+ *   };
+ * }} options
+ */
+export const startGovernedInstance = async (
+  {
+    consume: {
+      agoricNames,
+      board,
+      chainTimerService: chainTimerServiceP,
+      economicCommitteeCreatorFacet: electorateCreatorFacet,
+      governedContractKits: governedContractKitsP,
+      zoe,
+    },
+    produce,
+  },
+  {
+    options: { contractInstallation, producedKey, label = 'governedContract' },
+  },
+) => {
+  trace('Governed Contract start');
+
+  const poserInvitationP = E(electorateCreatorFacet).getPoserInvitation();
+  const [chainTimerService, poserInvitation, governedContractKits] =
+    await Promise.all([
+      chainTimerServiceP,
+      poserInvitationP,
+      governedContractKitsP,
+    ]);
+
+  const invitationIssuer = await E(zoe).getInvitationIssuer();
+  const invitationAmount =
+    await E(invitationIssuer).getAmountOf(poserInvitation);
+
+  const governedTerms = {
+    governedParams: {
+      [MALLEABLE_NUMBER]: {
+        type: ParamTypes.NAT,
+        value: 602214090000000000000000n,
+      },
+      [CONTRACT_ELECTORATE]: {
+        type: ParamTypes.INVITATION,
+        value: invitationAmount,
+      },
+    },
+  };
+
+  const governorTerms = {
+    chainTimerService,
+    governedContractInstallation: contractInstallation,
+    governed: {
+      terms: governedTerms,
+      issuerKeywordRecord: {},
+      label,
+    },
+  };
+
+  const governorInstallation = await E(agoricNames).lookup(
+    'installation',
+    'contractGovernor',
+  );
+
+  const governorStartResult = await E(zoe).startInstance(
+    governorInstallation,
+    {},
+    governorTerms,
+    {
+      // electorateCreatorFacet,
+      governed: {
+        initialPoserInvitation: poserInvitation,
+      },
+    },
+    `${label}.governor`,
+  );
+
+  const kit = await deeplyFulfilledObject(
+    harden({
+      label,
+      creatorFacet: E(governorStartResult.creatorFacet).getCreatorFacet(),
+      adminFacet: E(governorStartResult.creatorFacet).getAdminFacet(),
+      publicFacet: E(governorStartResult.creatorFacet).getPublicFacet(),
+      instance: E(governorStartResult.creatorFacet).getInstance(),
+
+      governor: governorStartResult.instance,
+      governorCreatorFacet: governorStartResult.creatorFacet,
+      governorAdminFacet: governorStartResult.adminFacet,
+    }),
+  );
+
+  governedContractKits.init(kit.instance, kit);
+
+  if (producedKey !== undefined) {
+    const boardID = await E(board).getId(kit.instance);
+    trace({ boardID });
+    produce[producedKey].reset();
+    produce[producedKey].resolve({ ...kit, boardID });
+  }
+};
+
+/**
+ * Add a new auction to a chain that already has one.
+ *
+ * @param {object} utils
+ * @param {any} utils.restoreRef
+ * @param {any} addAuctionOptions
+ */
```

Comment:
```suggestion
/**
 * Return a manifest for instantiating a new governed contract instance.
 *
 * @param {object} utils
 * @param {(b: ERef<ManifestBundleRef>) => Promise<Installation>} utils.restoreRef
 * @param {object} details
 * @param {ERef<ManifestBundleRef>} details.contractBundleRef source bundle for
 *   the new governed contract
 * @param {string} [details.producedKey] if present, the Bootstrap Powers key
 *   with which a reference to the `boardID`-enhanced governed contract kit
 *   should be associated
 * @param {string} [details.label] for identification
 */
```
with this somewhere near top of file:
```js
/**
 * @import {ManifestBundleRef} from '@agoric/deploy-script-support/src/externalTypes.js'
 */
```